ffv1.md

```diff
@@ -443,11 +443,14 @@ R~i~
 : the Range at the i-th symbol.
 
 r~i~
-: the boundary between two sub-ranges of R~i~: a sub-range of r~i~ values and a sub-range R~i~ - r~i~ values.
+: the Range at the i+1-th symbol. For R~i~ this represents the boundary between two sub-ranges of R~i~: a sub-range of r~i~ values and a sub-range R~i~ - r~i~ values.
 
 L~i~
 : the Low value of the Range at the i-th symbol.
 
+l~i~
+: the Low value of the Range at the i+1-th symbol.
+
```

Comment:
This could be confusing. it would imply l() and L() are the same and redundant

Reply:
written as `a temporary variable to carry-over or adjust the Low value of the Range between range coding operations.`